frontend/src/index.css

```diff
@@ -10,15 +10,6 @@
   --color-border: rgba(57, 53, 52, 0.125);
   --color-border-strong: rgba(57, 53, 52, 0.5);
   --color-error: #b8003f;
-  /* --- Test Colors */
```

Comment:
Ah yes, thought about removing them earlier. Thanks for removing all that clutter!